test/unit/test-atomic.cpp

```diff
@@ -42,6 +42,8 @@ void testAtomicFunctionBasic()
   dest[5] = (T)0;
   dest[6] = (T)N + 1;
   dest[7] = (T)0;
+  dest[8] = (T)N;
+  dest[9] = (T)0;
```

Comment:
Does the size of the new and cudaMalloc above need to become 10?

Reply:
good catch, fixed.